edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java

```diff
@@ -105,6 +111,60 @@ public static List<Slot> getSchedule(Group group, Interval timerange) {
         return schedule.filterEverySlotOutsideTimerange(allSlotOfGroup, timerange);
     }
 
+    /**
+     * Get Student of Database via email & password
+     * @param email Email of the student
+     * @param password password of the student
+     * @return Student entity if exist in database, otherwise return null.
+     */
+    public static Optional<Student> getStudentFromAuth(String email, String password){
+        StudentDAO dao = StudentDAO.getInstance();
+        Student student = null;
+
+        try {
+            student = dao.get(email, password).orElseThrow(SQLException::new);
```

Comment:
I think in the future we should hash the pwd in the facade, but of course only when we tackle #47